Standard/BisCustomAttributes.ecschema.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="BisCustomAttributes" alias="bisCA" version="01.00.00" description="Custom attributes to indicate BIS concepts." displayLabel="BIS Custom Attributes" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECEnumeration typeName="SchemaGradeValue" backingTypeName="string" isStrict="true" description="Used with SchemaType to declare the schema author's intended grade for the schema.">
+        <ECEnumerator displayLabel="A" value="A"/> <!-- description="True BIS schemas carefully designed for editing and interoperability." -->
+        <ECEnumerator displayLabel="B" value="B"/> <!-- description="Either: Legacy “consensus” schemas intelligently converted to BIS, or new BIS schemas, with one-way conversion to BIS in mind, but not intended for editing (native format)." -->
+        <ECEnumerator displayLabel="C" value="C"/> <!-- description="Legacy schema with software-discoverable semantics, intelligently converted to follow applicable BIS rules and patterns." -->
+        <ECEnumerator displayLabel="D" value="D"/> <!-- description="Legacy schema with minimim or no software-discoverable semantics, converted to follow basic BIS rules and patterns." -->
+    </ECEnumeration>
+
+    <ECEnumeration typeName="SchemaLayerValue" backingTypeName="string" isStrict="true" description="Used with SchemaType to declare the schema author's intended layer for the schema.">
+        <ECEnumerator displayLabel="Core"                value="Core"/>               <!-- description="Schemas defining the 'fabric of the universe' and some key organizational strategies." -->
+        <ECEnumerator displayLabel="Common"              value="Common"/>             <!-- description="Schemas defining abstract concepts and patterns used by multiple disciplines." -->
+        <ECEnumerator displayLabel="Discipline-Physical" value="DisciplinePhysical"/> <!-- description="Schemas defining physical/spatial and closely associated concepts, in light of a specific discipline." -->
+        <ECEnumerator displayLabel="Discipline-Other"    value="DisciplineOther"/>    <!-- description="Schemas defining concepts from modeling perspectives other than physical, in light of a specific discipline." -->
```

Comment:
I may have missed the discussion on this, but, was consideration made for the word "Domain" instead of "Discipline"?

Reply:
Yes. BIS docs in general treat "domain" and "schema" as pretty much synonymous. It was discussed that we will use "discipline" as the larger-scope term for one or more schemas, together, addressing an area of practice.